test/integration/routes/routes_test.go

```diff
@@ -82,19 +82,4 @@ func TestRoutes(t *testing.T) {
 				"expected %d occurrences of \"%s\", got %d\n%s", r.c, r.s, count, out)
 		}
 	}
-
-	// smoke test / bb routes
-	prefixedNs := TestHelper.GetTestNamespace("smoke-test")
```

Comment:
Are we deleting these assertions because the namespace no longer exists? Is the golden file for this specific route still used or can we clean that up as well?

Reply:
Yep, We can clean that too! We seem to run the routes test on the viz components so I don't see how this is useful especially now that these aren't available.

Reply:
Yeah, agreed it's not super useful, I'm actually glad to see this go! I think running the route test just on the viz components is enough. I also don't like that it was a bit too tightly coupled with the smoke tests which will make it harder to have independent/self-standing test suites.

This looks like a good change to do, would be great to clean up the unnecessary golden files with it too would be my only remark :)